### ISSUE TITLE
Why need dialog?

### ISSUE THREAD

**Issue by tomelsj** [hata6502/editorjs-layout#27]
Couldn't the editor be directly inside the layout instead of having a dialog popup? Or would it be some conflicts, for example it is difficult to know if it is the layout or the content inside I edit?

**Comment by hotrungnhan** [comment]
@hata6502  . I got an issue when using it with strapi. [strapi editor js](https://www.npm
<img width="1322" alt="Screenshot 2022-12-01 at 16 21 46" src="https://user-images.githubusercontent.com/40029631/205014968-1bf537bb-a668-471e-b540-0a26518e21a2.png">
js.com/package/strapi-plugin-react-editorjs)
I've implemented editors layout with strapi editor. But when I apply image tools to open strapi image picker pannel, this dialog top layer doesn't work as expect, i cant edit it z index for the editor to go below.
